drivers/tty/serial/uart-litex.c

```diff
@@ -228,12 +220,6 @@ static int litex_uart_startup(struct uart_port *port)
 	if (r == 0)
 		uart_out32(0x3, LITEX_UART_EV_ENABLE, port);
 
-#if 0 // TODO error
```

Comment:
Remove, since this enabled RX/TX and interrupts.  We don't need to enable RX/TX and we just enabled interrupts on the previous line.